res/values/strings.xml

```diff
@@ -33,13 +33,13 @@
     <string name="forward">Forward</string>
     <!-- "Resend" means "Sending the selected message(s) again to the same chat". The string is used in a menu and should be as short as possible. Resending may be needed after failures or to repost old messages to new members. -->
     <string name="resend">Resend</string>
-    <!-- In the meaning of "To archive", "Put a chat to the archive". This string does not mean "The Archive". -->
+    <!-- Verb "to archive", as in "put a chat in the archive", not a noun "The Archive". -->
     <string name="archive">Archive</string>
-    <!-- In the meaning of "To unarchive", "Remove a chat from the archive", opposite of the previous string -->
+    <!-- Verb "to unarchive", as in "remove a chat from the archive", opposite of the previous string -->
     <string name="unarchive">Unarchive</string>
     <string name="mute">Mute</string>
     <string name="ephemeral_messages">Disappearing Messages</string>
-    <string name="ephemeral_messages_hint">These settings apply to all chat members using Delta Chat. However, they may copy, save and forward messages or use other e-mail clients.</string>
+    <string name="ephemeral_messages_hint">Removes messages after the specified duration from all members of this chat; this applies only if they use Delta Chat and does not prevent copying, saving, and forwarding messages or using other e-mail clients.</string>
```

Comment:
tbh,i like the old sentence more. there was quite some effort put into make it that short and precise (place is limited!) 

eg. it is pretty clear by the context that it is about "removing messages after some duration".

Reply:
Now one character shorter than the original